save-backend/src/test/kotlin/org/cqfn/save/backend/controller/CloningRepositoryControllerTest.kt

```diff
@@ -76,7 +87,44 @@ class CloningRepositoryControllerTest {
             .isEqualTo("Clone pending")
     }
 
+    @Test
+    fun checkNewJobResponseForBin() {
+        val binFile = File(binFilePath)
+        val property = File(propertyPath)
+        val project = Project("noname", "1", "1", "1")
+        val request = BinaryExecutionRequest(project, emptyList())
+        val bodyBuilder = MultipartBodyBuilder()
+        bodyBuilder.part("binaryExecutionRequest", request)
+        bodyBuilder.part("property", property)
+        bodyBuilder.part("binFile", binFile)
+
+        mockServerPreprocessor.enqueue(
+            MockResponse()
+                .setResponseCode(202)
+                .setBody("Clone pending")
+                .addHeader("Content-Type", "application/json")
+        )
+
+        webTestClient.post()
+            .uri("/submitExecutionRequestBin")
+            .contentType(MediaType.MULTIPART_FORM_DATA)
+            .body(BodyInserters.fromMultipartData(bodyBuilder.build()))
+            .exchange()
+            .expectStatus()
+            .isEqualTo(HttpStatus.ACCEPTED)
+            .expectBody<String>()
+            .isEqualTo("Clone pending")
+    }
+
+    @AfterAll
+    fun removeBinDir() {
+        File(BIN_FOLDER).deleteRecursively()
+    }
+
     companion object {
+        const val BIN_FOLDER = "binFolder"
```

Comment:
Is it created in working directory of test runner? This would once again pollute working tree with junk files. You should use temporary locations for such things.